From 79726952ae4059b257a63451ff69da30da620530 Mon Sep 17 00:00:00 2001
From: Marine Vovard <m.vovard@phytec.de>
Date: Fri, 14 Jul 2023 14:39:20 +0200
Subject: [PATCH] Remove bad warning in accordion

---
 kivy/uix/accordion.py | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/kivy/uix/accordion.py b/kivy/uix/accordion.py
index fa855f4ed..6ed9b3fc2 100644
--- a/kivy/uix/accordion.py
+++ b/kivy/uix/accordion.py
@@ -405,13 +405,13 @@ class Accordion(Widget):
         else:
             display_space = self.height - min_space_total
 
-        if display_space <= 0:
-            Logger.warning('Accordion: not enough space '
-                           'for displaying all children')
-            Logger.warning('Accordion: need %dpx, got %dpx' % (
-                min_space_total, min_space_total + display_space))
-            Logger.warning('Accordion: layout aborted.')
-            return
+        #if display_space <= 0:
+            #Logger.warning('Accordion: not enough space '
+            #               'for displaying all children')
+            #Logger.warning('Accordion: need %dpx, got %dpx' % (
+            #    min_space_total, min_space_total + display_space))
+            #Logger.warning('Accordion: layout aborted.')
+            #return
 
         if orientation == 'horizontal':
             children = reversed(children)
-- 
2.25.1

