diff --git a/kivy/core/clipboard/__init__.py b/kivy/core/clipboard/__init__.py
index 2fe4a9e8e..1d218af9f 100644
--- a/kivy/core/clipboard/__init__.py
+++ b/kivy/core/clipboard/__init__.py
@@ -141,17 +141,17 @@ _clipboards.append(
 Clipboard = core_select_lib('clipboard', _clipboards, True)
 CutBuffer = None
 
-if platform == 'linux':
-    _cutbuffers = [
-        ('xclip', 'clipboard_xclip', 'ClipboardXclip'),
-        ('xsel', 'clipboard_xsel', 'ClipboardXsel'),
-    ]
-
-    if Clipboard.__class__.__name__ in (c[2] for c in _cutbuffers):
-        CutBuffer = Clipboard
-    else:
-        CutBuffer = core_select_lib('cutbuffer', _cutbuffers, True,
-                                    basemodule='clipboard')
-
-    if CutBuffer:
-        Logger.info('CutBuffer: cut buffer support enabled')
+#if platform == 'linux':
+#    _cutbuffers = [
+#        ('xclip', 'clipboard_xclip', 'ClipboardXclip'),
+#        ('xsel', 'clipboard_xsel', 'ClipboardXsel'),
+#    ]
+#
+#    if Clipboard.__class__.__name__ in (c[2] for c in _cutbuffers):
+#        CutBuffer = Clipboard
+#    else:
+#        CutBuffer = core_select_lib('cutbuffer', _cutbuffers, True,
+#                                    basemodule='clipboard')
+#
+#    if CutBuffer:
+#        Logger.info('CutBuffer: cut buffer support enabled')
